privacyidea/lib/eventhandler/tokenhandler.py

```diff
@@ -236,7 +237,8 @@ def actions(cls):
                                    "type": "str",
                                    "required": True,
                                    "description": _("Set the failcounter of "
-                                                    "the token.")
+                                                    "the token. Any value or increment n, "
```

Comment:
I am a bit confused by this sentence. "Any value or increment n" sounds like as if "n" would be an increment. May some more text might be less confusing like

"Increase or decrease or set to a specific value by ... doing/puttin/verb-of-your-choice +n, -n or n with n being an integer..."
..or...  rtfm: read the fantactic manual ;-)

Reply:
After pushing, I also wondered if the word order is misleading. Changed.